python-lib/wordcloud_visualizer.py

```diff
@@ -208,6 +211,27 @@ def _tokenize_texts(self, df_grouped: List) -> List:
 
         return docs
 
+    def _normalize_case_token_counts(self, counts: Counter) -> Counter:
+        """Private method to normalize a token counter to make it case-insensitive
+
+        It works by summing the counts of multiple case versions e.g., "The" and "the".
+        It then selects the case version with the highest count to represent the whole group.
+        In case of a tie, the lowercase version is chosen.
+
+        Args:
+            counts: token counter e.g., Counter({"The": 3, "the": 5, "best": 3, "Best": 4})
+        Returns:
+            New token counter with a single case version for each token and the sum of counts across case versions
+                e.g., Counter({"the": 8, "Best": 7})
+        """
+        df_counts = pd.DataFrame.from_dict(counts.items())
```

Comment:
Cf. comment on pandas 1 in code env desc.json and version support

Reply:
Yes, this is pandas 1.0 only. I guess that's fine as we won't support DSS 7 much longer